pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateData.java

```diff
@@ -27,7 +29,8 @@
  */
 
 public record ServiceUnitStateData(
-        ServiceUnitState state, String dstBroker, String sourceBroker, boolean force, long timestamp, long versionId) {
+        ServiceUnitState state, String dstBroker, String sourceBroker,
+        Map<String, Optional<String>> splitServiceUnitToDestBroker, boolean force, long timestamp, long versionId) {
```

Comment:
1. Don't we need to keep the order of boundaries? Can it be a List?

2. Can you explain why we have the dstBroker value? Can we remove the dstBroker? 

3. Currently, I think we only require split boundaries. Can we make it `List<Long> splitBoundaries`?

Reply:
> Don't we need to keep the order of boundaries? Can it be a List?

No, we don't need to keep the order of boundaries. Yes, it can be a list, but it cannot be duplicated.

> Can you explain why we have the dstBroker value? Can we remove the dstBroker? 

Do we need to assign the sub bundle directly to a dst broker and unload the parent bundle? If we need this feature, it can be impl in the future.

> Currently, I think we only require split boundaries. Can we make it List<Long> splitBoundaries?

If we need to support sub bundles direct assign, then we need to keep the map.

Reply:
Ok.